vivisect/analysis/__init__.py

```diff
@@ -55,9 +55,11 @@ def addAnalysisModules(vw):
         # Snap in an architecture specific emulation pass
         if arch == 'i386':
             vw.addFuncAnalysisModule("vivisect.analysis.i386.calling")
+            vw.addFuncAnalysisModule("vivisect.analysis.i386.instrhook")
```

Comment:
With how similar the i386 one and the amd64 one are, I would say just delete one of them and just reuse it for both i386 and amd64.

There may come a day where we split them, but today is not that today.